app/models/item.rb

```diff
@@ -0,0 +1,7 @@
+class Item < ApplicationRecord
+	belongs_to :category
+	belongs_to :sub_category
+	has_many   :order_items
+	has_many   :line_items
+	validates  :name, :price, :quantity, presence: true
```

Comment:
item's price should always be greater than or equal to 0
item's quantity should always be greater than to 0

Reply:
done